.github/workflows/push.yml

```diff
@@ -14,7 +14,7 @@ jobs:
     
     strategy:
       matrix:
-        python-version: ['3.7', '3.9']
+        python-version: ['3.7', '3.10']
```

Comment:
Any reason not to list all of 3.7, 3.8, 3.9, 3.10?

Reply:
Ah, perhaps because each one adds 6 more tests for java/os combinations?

Reply:
Yes. I think we only need to test the extremes.